sky/authentication.py

```diff
@@ -10,6 +10,8 @@
 from typing import Any, Dict, Tuple
 import uuid
 
+from urllib.parse import urlparse
```

Comment:
nit: since urllib is standard library, perhaps should be added at L11 before `import uuid`.

Reply:
Hi @romilbhardwaj , I removed `import urlparse`, as it is now being used by `clouds.Kubernetes.get_external_ip()` at `setup_kubernetes_authentication`.